src/nwb2bids/_command_line_interface/_main.py

```diff
@@ -62,9 +62,18 @@ def _run_convert_nwb_dataset(
         raise ValueError(message)
     handled_nwb_paths = [pathlib.Path(nwb_path) for nwb_path in nwb_paths]
 
-    convert_nwb_dataset(
+    messages = convert_nwb_dataset(
         nwb_paths=handled_nwb_paths,
         bids_directory=bids_directory,
         file_mode=file_mode,
         additional_metadata_file_path=additional_metadata_file_path,
     )
+
+    if messages is not None:
```

Comment:
QUESTION: should we add a `--save-messages` flag to save a `.json` dump of the messages in the home folder (like a log file)?

Reply:
SUGGESTION FROM YARIK: Yes, just save them, treat like 'log files'